modules/virt-configuring-ip-for-vm.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
+
+[id="virt-configuring-ip-for-vm_{context}"]
+= Configuring an IP address for a virtual machine
+
+You can use cloud-init to configure an IP address for a virtual machine. The IP address can be provisioned to be dynamic or static. 
+
+You can add or edit the cloud-init configuration of an existing virtual machine configuration. If you edit a running virtual machine, the virtual machine must be rebooted for the changes to take effect.
+
+.Prerequisites
+
+* DNS is configured via DHCP on the cluster.
+
+* A Linux bridge network attachment definition.
+
+* The virtual machine is connected to the bridge network. 
+
+// web console and CLI? Link to edit in web console?
```

Comment:
What's up with these comments?

Reply:
Note to myself while drafting. The procedure isn't specific to web console or CLI but our docs normally are. I've tried to keep this pretty agnostic as to whether it's in web or CLI and edit or new but still weighing up whether I should mention and/or link to the edit VM docs - but didn't want that to delay review.